src/app.js

```diff
@@ -14,15 +19,36 @@ const postRoutes = require('./routes/post');
 const app = express();
 const port = process.env.NODE_LOCAL_PORT || 3000;
 
-app.use(bodyParser.json());
+app.use(cookieParser(process.env.SECRET_KEY));
+app.use(encryptCookieNodeMiddleware(process.env.SECRET_KEY));
 
-app.use('/api/auth', authRoutes);
-app.use('/api/user', userRoutes);
-app.use('/api/post', postRoutes);
+const publicPaths = [
+  '/api/',
+  '/api/about',
+  '/api/auth/login',
+  '/api/auth/register',
+];
```

Comment:
since this middleware is gonna be used on all the endpoints can you add the other public endpoints to the array, like `/`, `/api/auth/google`, `/api/auth/google/callback`, `/api/auth/facebook`, `/api/auth/facebook/callback` and put it in the constant file.
![image](https://user-images.githubusercontent.com/98318641/182239449-d9e775b2-661d-4728-bdfb-563114f44b0f.png)

Reply:
sure